fud2/fud-core/src/exec/driver.rs

```diff
@@ -455,29 +469,52 @@ impl DriverBuilder {
     }
 }
 
+/// A file tagged with it's input source.
```

Comment:
```suggestion
/// A file tagged with its input source.
```

To be a little more general (because it's not just for inputs): A filename used in a plan. Files can either be "ordinary" files on disk or ones associated with stdin/stdout streams.